fairing/deployers/serving/serving.py

```diff
@@ -0,0 +1,82 @@
+import json
+import uuid
+import logging
+
+from kubernetes import client as k8s_client
+from fairing.deployers.job.job import Job
+
+logger = logging.getLogger(__name__)
+
+class Serving(Job):
+    """
+    Serves a prediction endpoint using Kubernetes deployments and services 
+    
+    serving_class: the name of the class that holds the predict function.
+    
+    """
+    def __init__(self, serving_class, namespace=None, runs=1, ):
+        super(Serving, self).__init__(namespace, runs)
+        self.serving_class = serving_class
+
+    def create_resource(self):
```

Comment:
Do we need this in the Serving deployer? I don't see where it's being used.

Reply:
good catch - not used. deleted